src/mococrw/key.h

```diff
@@ -270,7 +271,55 @@ class AsymmetricKeypair : public AsymmetricPublicKey
      * Loads a private key from an HSM, creating an @ref AsymmetricPublicKey
      * object as a result.
      */
-    static AsymmetricKeypair readPrivateKeyFromHSM(HSM &hsm, const std::string &keyID);
+    static AsymmetricKeypair readPrivateKeyFromHSM(const HSM &hsm, const std::string &keyID);
```

Comment:
Why is the key identified by tokenLabel, keyID and keyLabel in the generation API, but here only KeyID is used?

Would be cool to consolidate and minimize the number of artifacts you need to identify one key pair.